src/Application/index.js

```diff
@@ -119,6 +119,22 @@ export default function(App, appData, platformSettings) {
       Accessibility.colorshift(this, type, config)
     }
 
+    get keymapping() {
+      return this.stage.application.config.keys
+    }
+
+    set keymapping(overrideKeymap) {
+      const baseKeymap = this.stage.application.config.keys
+      Object.keys(overrideKeymap).reduce((keymapping, key) => {
+        // prevent duplicate values to exist in final keymapping (i.e. 2 keys listening to 'Back')
+        const duplicateIndex = Object.values(baseKeymap).indexOf(overrideKeymap[key])
```

Comment:
It is currently possible to define 2 different keys listening to 'Back' by using `settings.json`. Using 2 different keys for the same handler is also referenced in [src/Application/index.js](https://github.com/rdkcentral/Lightning-SDK/blob/2b0d10e6d36ae9d855d8f0ac29e8eaccae807ebf/src/Application/index.js#L55). 

I think it could be better to reconsider removing duplicated values here while overriding keys because duplicated values might be coming from `settings.json`.

Reply:
As an alternative way, `overrideKeymap` can completely replace the existing `baseKeymap` and the decision of merging key map can be left to developers. Because with the overriding feature they can provide any key map (either merged with the base key map or a completely new key map) based on their needs.

Reply:
As discussed offline: we’ll keep the default behaviour as is. We will however introduce a flag to enable keeping defaults (or disable the removal of defaults, whichever way way of looking at it is more logical).

This means that the setter will have to become a function (which makes more sense anyway) in order to accept the option flag.